### DISCUSSION TITLE
Wagmi has potential but still far away

### DISCUSSION THREAD

**Discussion by kwatts22** [wevm/wagmi#3671] [General]
While the framework seems promising, the documentation sucks. It bounces around so much that it makes things very confusing to learn and a lot of code is missing in the hooks that you need.  For now, I will continue to use ethers.js.

**Comment by jxom** [comment]
Thanks for the feedback, I'll pass it on to the team.

**Comment by tmm** [comment]
Hi! Senior Project Manager (Documentation) on the Wagmi Team here. If you can provide more details on the documentation that "sucks", we'd be happy to discipline the engineers responsible for writing such bad documentation. Looking forward to hearing back!

**Reply by kwatts22** [reply]
The Wallet Connect won't open in Microsoft Edge. Having all kinds for troubles reading the contract. Get all kinds of error in visual studio.

**Reply by tmm** [reply]
Really appreciate this feedback. 

While waiting for your reply, we decided to take expedited action due to the seriousness of these issues. The three engineers responsible for the documentation website have been terminated without benefits effective immediately.

While we hire another documentation team, our intern, @jxom, will be taking over documentation duties. We look forward to improving our documentation so this doesn't happen again.

**Reply by jxom** [reply]
I am on it.

**Reply by lochie** [reply]
rip docs team, they will be missed

**Reply by colinnielsen** [reply]
🪦 vocs

**Reply by boidushya** [reply]
I knew one of the three people who got fired, good lad 😔

**Reply by yura2100** [reply]
> While waiting for your reply, we decided to take expedited action due to the seriousness of these issues. The three engineers responsible for the documentation website have been terminated without benefits effective immediately.
> 
> While we hire another documentation team, our intern, @jxom, will be taking over documentation duties.

I am not sure this would help. I believe the only right decision is to close wagmi and just admit that web3.js is the only right way to interact with Ethereum. Unfortunately viem should be closed as well 😢

**Reply by ghost** [reply]
I have 10+ years of professional coding experience but I am new to React and Wagmi (and website and blockchain development as a whole) and I can confirm that I had a hard time figuring out how to use the Wagmi hooks as the documentation seems to be written with the assumption that the reader has advanced knowledge in all the relevant technologies/frameworks/tools/etc. Also the blurring on the code examples looks and feels really bad

**Reply by strollinghome** [reply]
Skill issues imho

**Reply by ghost** [reply]
@strollinghome you can be ignorant all you want, I dont care. Trying to give an additional perspective into the discussion that may help to make the ecosystem more userfriendly. A alot of people are clearly stating that the docs are not good enough, sadly seems they/we are all just not as "smart" as you

**Reply by tmm** [reply]
Some people love the docs and some people hate them. We get mixed feedback. Feel free to suggestion changes to them!

**Comment by kwatts22** [comment]
Firing them doesn't solve the issues. Learning and building on it to make wagmi better is the best way.

**Comment by angryhayder** [comment]
i ve been using wagmi since the early versions, when you guys finally reached a good stable release in v1 , viem messed things up, it's not your mistake guys , just the docs aren't as straightforward as they were , @tmm you re joking about terminating them but i think most people should be advised to stick to v1.x for now . seemless integration with rainbowkit etc , it's still miles better than row eth directly , just keep up the good work guys !
